dts/arm/ti/mspm0g1xxx.dtsi

```diff
@@ -110,14 +110,14 @@
                 };
 
 		tima0: tima0@40860000 {
-                    compatible      = "ti,mspm0-pwm";
-                    status          = "disabled";
-                    ti,clock-frequency = <32000000>;
-                    reg                = <0x40860000 0x2000>;
-                    ti,divide-ratio    = <RATE_1>;
-                    ti,prescaler       = <0>;
-                    ti,mode            = <EDGE_ALIGN_UP>;
-		    #pwm-cells 	       = <3>;
+                    compatible = "ti,mspm0-pwm";
+                    status = "disabled";
+                    clocks = <&sysclk>;
+                    reg = <0x40860000 0x2000>;
+                    ti,divide-ratio = <RATE_1>;
+                    ti,prescaler = <0>;
+                    ti,mode = <EDGE_ALIGN_UP>;
+                    #pwm-cells = <3>;
```

Comment:
Formatting is off

Reply:
formatting was off and this change fixed it, not sure why it shows like that